src/chess/net/net_trivial_test.rs

```diff
@@ -0,0 +1,29 @@
+#[cfg(test)]
+mod tests {
+    use crate::chess::chess_game::ChessPosition;
+    use crate::chess::net::net_trivial::TrivialNet;
+    use crate::game::mcts::ValueFunction;
+
+    #[test]
+    fn basic_evaluate() {
+        let mut net = TrivialNet::new();
+
+        let pos1 = ChessPosition::from_str(
+            &"rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1".to_string(),
+        );
+        let pos2 = ChessPosition::from_str(
+            &"rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/R1BQKBNR w KQkq - 0 1".to_string(),
+        );
+        let pos3 = ChessPosition::from_str(
+            &"rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR b KQkq - 0 1".to_string(),
+        );
+        let pos4 = ChessPosition::from_str(
+            &"r1bqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR b KQkq - 0 1".to_string(),
+        );
+
+        assert!(net.evaluate(&pos1).0 == -net.evaluate(&pos3).0);
+        assert!(net.evaluate(&pos2).0 == -net.evaluate(&pos4).0);
+        assert!(net.evaluate(&pos1).0 > net.evaluate(&pos2).0);
+        assert!(-net.evaluate(&pos3).0 > -net.evaluate(&pos4).0);

```

Comment:
is there any board that you know its numerical value? some test other than a==b and a>b 
even create a board with one piece, or an almost-symmetric board

Reply:
I prefer not to do so, this test only test that the flipping, feeding and reading of the positions values and probabilities are making sense, I don't want to be strick about the test, we can use this test for the Stockfish network as well

Reply:
so maybe there should be a different test that checks some actual numerical values? 
to make sure that the calculations do what you expect them to do